sechub-solution/docker/install-java/fedora/install-java.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env sh
+# SPDX-License-Identifier: MIT
+
+print_error() {
+    message="$1"
+
+    echo "$message" 1>&2
+}
+
+JAVA_DISTRIBUTION="$1"
+JAVA_VERSION="$2"
+JAVA_RUNTIME="$3"
+
+JAVA_DIR="/opt/java"
+
+if [ -z "$JAVA_DISTRIBUTION" ]
+then
+    print_error "ERROR: No Java distribution provided!"
+    exit 1
+fi
+
+if [ -z "$JAVA_VERSION" ]
+then
+    print_error "ERROR: No Java version provided!"
+    exit 1
+fi
+
+if [ -z "$JAVA_RUNTIME" ]
```

Comment:
If the possible values are jre and jdk, you could check this
```suggestion
if [ "$JAVA_RUNTIME" != "jdk" && "$JAVA_RUNTIME" != "jre" ]
```